web/source/text/outputTarget.ts

```diff
@@ -122,46 +129,46 @@ namespace com.keyman.text {
       return new TextTransform(delta, deletedLeft, originalRight - undeletedRight);
     }
 
-    buildTranscriptionFrom(original: OutputTarget, keyEvent: KeyEvent): Transcription {
+    buildTranscriptionFrom(original: OutputTarget, keyEvent: KeyEvent, alternates?: Alternate[]): Transcription {
       let transform = this.buildTransformFrom(original);
 
-      // While not presently needed, there's a chance we'll want to have Deadkey mutation tracked
-      // for developments later down the line.  May as well resolve the logic now.
+      // // While not presently needed, there's a chance we'll want to have Deadkey mutation tracked
```

Comment:
I would just remove this entirely with a commit message saying "Delete Deadkey mutation". Then just revert the commit that deleted the deadkey mutation later when we _do_ need it.